lib/database_cleaner/active_record/deletion.rb

```diff
@@ -66,8 +66,9 @@ def table_stats_query(connection, db_name)
       if @cache_tables && !@table_stats_query.nil?
         return @table_stats_query
       else
+        quote_start, quote_end = connection.quote_table_name('T').split('T')
         @table_stats_query = connection.select_values(<<-SQL).join(' UNION ')
-               SELECT CONCAT('SELECT \"', table_name, '\" AS table_name, COUNT(*) AS exact_row_count FROM ', table_name)
+               SELECT CONCAT('SELECT \"', table_name, '\" AS table_name, COUNT(*) AS exact_row_count FROM #{quote_start}', table_name, '#{quote_end}')
```

Comment:
@odedniv This implementation looks more complicated than it needs to be. Could you see how this was implemented here: https://github.com/DatabaseCleaner/database_cleaner/blob/master/lib/database_cleaner/active_record/deletion.rb#L17

Is there a good reason you didn't just use `quote_table_name`?

Reply:
Generating the SQL strings in the database is really ugly. :smile: If you pull that bit into Ruby land you don't have trouble quoting properly, e.g. like this: #501

Reply:
Agreed, if you're OK with generating the SQL in Ruby (not sure about the reasoning behind CONCATing in SQL), I prefer #501 and this PR can be closed.